terraform/Mo/rancher-master/keypairs.tf

```diff
@@ -0,0 +1,7 @@
+resource "openstack_compute_keypair_v2" "rancher_node_keypair" {
+  name = "rancher-node"
+}
+
+resource "openstack_compute_keypair_v2" "rancher_master_keypair" {
```

Comment:
This is a security risk, as by doing that the master keypair will be written into the statefile for everyone to read plain